### DISCUSSION TITLE
CouchDB Backup

### DISCUSSION THREAD

**Discussion by SimonBeau** [vrtmrz/obsidian-livesync#345] [General]
Hello!

Thank you for this plugin. It's working excellently for me across many devices :)

I've set up with CouchDB running in a container on my NAS. I'm curious if you have any recommendations for doing regular backups? 

Cheers

**Comment by dlphnkck** [comment]
Hi. I am not familiar with CouchDB, but it seems that there is a way to do the following. I don't know if this will be helpful.

1. CouchDB Replication 
[5\.3\. Backing up CouchDB — Apache CouchDB® 3\.3 Documentation](https://docs.couchdb.org/en/stable/maintenance/backups.html)
> The simplest and easiest approach for CouchDB backup is to use CouchDB replication to another CouchDB installation. 

2. CouchBackup
[Backing up and restoring CouchDB data \- IBM Documentation](https://www.ibm.com/docs/en/memao/1.1.0?topic=data-backing-up-restoring-couchdb)
> Use CouchBackup to back up and restore CouchDB data. CouchBackup is a command-line utility that allows a CouchDB database to be backed-up to a text file.

That is all. I hope this is of some help.

**Reply by vrtmrz** [reply]
Thank you for the follow-up! This is the ideal way to do it.
However, we can always rebuild the remote database. We have files which have been synchronised.
